processor/transformprocessor/factory.go

```diff
@@ -46,7 +47,7 @@ func createDefaultConfig() config.Processor {
 		Traces: TracesConfig{
 			Queries: []string{},
 
-			functions: traces.DefaultFunctions(),
+			functions: common.DefaultFunctions(),
```

Comment:
Should this remain a reference to `traces`?  Is it likely in the future that there will be default functions that apply only to traces, or only to metrics?  It could remain a passthrough to `common.DefaultFunctions()` for now, but still make clear the extension point exists.

Reply:
Sounds good